pkgs/top-level/haskell-packages.nix

```diff
@@ -462,6 +462,20 @@ in
         llvmPackages = pkgs.llvmPackages_15;
       };
       ghc910 = compiler.ghc9101;
+      ghc9121 = callPackage ../development/compilers/ghc/9.12.1.nix {
+        bootPkgs =
+          # No suitable bindist packaged yet
+          bb.packages.ghc9101;
+        inherit (buildPackages.python3Packages) sphinx;
+        # Need to use apple's patched xattr until
+        # https://github.com/xattr/xattr/issues/44 and
+        # https://github.com/xattr/xattr/issues/55 are solved.
+        inherit (buildPackages.darwin) xattr autoSignDarwinBinariesHook;
+        # 2023-01-15: Support range >= 11 && < 16
+        buildTargetLlvmPackages = pkgsBuildTarget.llvmPackages_18;
```

Comment:
According to the user guide it is now "between 13 and 20"
https://downloads.haskell.org/ghc/9.12.1/docs/users_guide/phases.html#ghc-flag-fllvm

Reply:
Thanks, updated.